@ecomplus/storefront-renderer/functions/index.js

```diff
@@ -26,6 +28,16 @@ exports.ssr = (req, res, getCacheControl) => {
       )
   }
 
+  const proxy = url => {
+    const urlInstance = new URL(url)
+    const requestUrl = urlInstance.searchParam.get('url')
+    if (requestUrl) {
+      return axios.get(requestUrl, { headers, timeout: 3000 })
+        .catch(console.error)
```

Comment:
O usuário envia algo errado ou o backend retorna algo errado, você salva em `console.error` mesmo não sendo um erro da function e não retorna nada pro usuário?

Reply:
É isso que tá acontecendo aqui 🙃

Reply:
Não é pra bater nada em console, é um erro do usuário ou do backend que o usuário tá tentando, o proxy (novamente) é só uma ponte, ele tem que retornar o erro ao usuário...

Reply:
Se o backend retorna um erro (a.k.a. 5xx) o proxy tem que repassar o erro "as is", se o erro for do usuário (URL inválido por exemplo) e o axios retornar der erro (não vai ter resposta) seu proxy deve retornar um 400 pro cliente (usuário), ele fez algo de errado

Reply:
Retornar tipo 400 e uma informação no data?

Reply:
O correto é especificar o que tá errado, mas se você retornar 400 quando for erro do cliente já tá valendo

Reply:
Vou facilitar proc, dá uma olhada no `validateStatus` aí https://axios-http.com/docs/req_config

Reply:
Muda isso pra sempre retornar true, como é só um proxy reverso não é pra "invalidar" a resposta do backend independente do status, se o backend responder algo pro proxy tá tudo certo, só repassar..